### ISSUE TITLE
route 53 record update no longer working

### ISSUE THREAD

**Issue by igg** [StackExchange/dnscontrol#402]
We are using gcloud and aws as redundant providers. Recently, when updating an A record for a subdomain, the Route53 update started failing, while the gcloud continued to work.
The error was (with domain names and IP numbers changed):
```----- DNS Provider: r53_main...1 correction
#1: MODIFY A subd1.example.com: (12.34.56.78 ttl=300) -> (12.34.56.89 ttl=300)

FAILURE! InvalidChangeBatch: RRSet with DNS name subd1. is not permitted in zone example.com.
	status code: 400, request id: 12345678-1234-1234-1234-123456789012
```
I checked that the same aws credentials were able to change this record using [cli53](https://github.com/barnybug/cli53), after which `dnscontrol preview` confirmed that the record no longer needed changing. We've used dnscontrol to change this record several times previously without issue. The version used was `dnscontrol 0.2.6-dev`

**Comment by igg** [comment]
Well, this is slightly embarrassing.  It turns out that the released version 0.2.6 works fine. The problem was using `go get github.com/StackExchange/dnscontrol`

**Comment by liljenstolpe** [comment]
I'm seeing the same behavior, but it actually looks like the record is installed in AWS, it's just also throwing the failure.  Odd.  Also on 0.2.6-dev (to get hexonet).
